Android.mk

```diff
@@ -131,7 +127,11 @@ LOCAL_SRC_FILES := \
     src/dbus/server/dbus_thread_object.cpp \
     src/dbus/server/error_helper.cpp \
     src/mdns/mdns.cpp \
+    src/sdp_proxy/advertising_proxy.cpp \
+    src/sdp_proxy/discovery_proxy.cpp \
     src/utils/hex.cpp \
+    src/utils/ncp_openthread.cpp \
```

Comment:
```suggestion
    src/ncp/ncp_openthread.cpp \
```

move to ncp ?

Reply:
From fuchsia's perspective, we will need to include //src/utils anyway, so I did not add another directory for ncp. I am ok with adding a //src/ncp if that looks better to you.